website/src/pages/leaderboard/score-leaderboard.tsx

```diff
@@ -0,0 +1,53 @@
+import RankItem from "@/components/LeaderBoardItem";
```

Comment:
For the URL, maybe we can just call this `leaderboard/score.tsx`.  I'm not sure we need to have leaderboard twice in the url.